codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -346,29 +370,62 @@ class FluentClientGenerator(
                         Ok(#{TypedBox}::<#{OperationOutput}>::assume_from(output).expect("correct output type").unwrap())
                     }
                     """,
-                    "HttpResponse" to RuntimeType.smithyRuntimeApi(runtimeConfig).resolve("client::orchestrator::HttpResponse"),
+                    "HttpResponse" to RuntimeType.smithyRuntimeApi(runtimeConfig)
+                        .resolve("client::orchestrator::HttpResponse"),
                     "OperationError" to errorType,
                     "Operation" to symbolProvider.toSymbol(operation),
                     "OperationOutput" to outputType,
-                    "RuntimePlugins" to RuntimeType.smithyRuntimeApi(runtimeConfig).resolve("client::runtime_plugin::RuntimePlugins"),
+                    "RuntimePlugins" to RuntimeType.smithyRuntimeApi(runtimeConfig)
+                        .resolve("client::runtime_plugin::RuntimePlugins"),
                     "SdkError" to RuntimeType.sdkError(runtimeConfig),
                     "TypedBox" to RuntimeType.smithyRuntimeApi(runtimeConfig).resolve("type_erasure::TypedBox"),
                     "invoke" to RuntimeType.smithyRuntime(runtimeConfig).resolve("client::orchestrator::invoke"),
                 )
-            }
-            PaginatorGenerator.paginatorType(codegenContext, generics, operation, retryClassifier)?.also { paginatorType ->
+
                 rustTemplate(
                     """
-                    /// Create a paginator for this request
+                    /// Sets the `config_override` for the builder.
+                    ///
+                    /// `config_override` is applied to the operation configuration level.
+                    /// For the fields in the builder that are not `None, they override those applied to the service
+                    /// configuration level or the AWS configuration level.
```

Comment:
Can't talk about the SDK in the generic client docs since it won't make sense for generic client customers.
```suggestion
                    /// configuration level.
```

Reply:
Ah, good point.